app/models/api/openid_connect/o_auth_application.rb

```diff
@@ -0,0 +1,62 @@
+module Api
+  module OpenidConnect
+    class OAuthApplication < ActiveRecord::Base
+      has_many :authorizations, dependent: :destroy
+      has_many :user, through: :authorizations
+
+      validates :client_id, presence: true, uniqueness: true
+      validates :client_secret, presence: true
+      validates :client_name, presence: true
```

Comment:
@theworldbright: can you also add `validates (:redirect_uris, :client_name), uniqueness: true`, regarding what we discussed yesterday?